runeq/v2sdk/stream.py

```diff
@@ -0,0 +1,60 @@
+"""
+V2 SDK functionality to support Stream operations.
+
+"""
+from runeq import errors
+from . import client
+
+
+def get_stream(stream_id: str, **params):
+    """
+    Fetch stream data from the Stream API.
+
+    Parameters
+    ----------
+    stream_id : str
+        Unique string identifier for a stream.
+
+    **params
+        limit: Limits the number of points returned for the stream.
+        format: 'csv' or 'json'. Defaults to csv.
```

Comment:
Maybe this has already been discussed, but as a UX consideration, I do wonder if `csv` is really the reasonable default for the SDK to use, but maybe it's common to pass string blobs containing CSV data to analysis libraries?

Reply:
I picked csv as the default because our internal neuro team transform the data they pull from the V1 API into pandas dataframes immediately before working with it. Pandas has a way to easily convert a csv into a dataframe, so it would be more efficient to store it as a csv.

Reply:
Ah cool, makes sense!